packs/spell-effects/spell-effect-charged-javelin-attacker.json

```diff
@@ -0,0 +1,56 @@
+{
+    "_id": "aNomC4orI44YGBGU",
+    "img": "systems/pf2e/icons/spells/charged-javelin.webp",
+    "name": "Spell Effect: Charged Javelin (Attacker)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Charged Javelin]</p>\n<p>Creatures gain a +1 status bonus to attack rolls with metal weapons or electricity effects against the target.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Lost Omens: Gods & Magic"
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    {
+                        "or": [
+                            "item:trait:electricity",
+                            {
+                                "and": [
+                                    "item:material:metal",
```

Comment:
We can at least do an or with all the known metal types. adamantine, cold iron, etc